src/v2/core.ts

```diff
@@ -112,3 +112,99 @@ export interface CloudFunction<EventType extends CloudEvent<unknown>> {
    */
   run(event: EventType): any | Promise<any>;
 }
+
+/**
+ * The Functions interface for events that change state, such as
+ * Realtime Database or Cloud Firestore `onWrite` and `onUpdate`.
+ *
+ * For more information about the format used to construct `Change` objects, see
+ * [`cloud-functions.ChangeJson`](/docs/reference/functions/cloud_functions_.changejson).
```

Comment:
This seems like a wrong link yes?

Reply:
Updated the TSDOC to just link to `ChangeJson`